src/Alexa.php

```diff
@@ -1,4 +1,6 @@
-<?php namespace Develpr\AlexaApp;
```

Comment:
Thanks for this. I don't remember WHEN, but a while back I believe this was something of a "best practice" that somebody put forth. Maybe it wasn't *actually* a best practice, but somewhere I read something that stuck in my head that this was the right way to do things. So it's been a habit of sorts. I just went through a few random Laravel and Symfony classes and sure enough, they don't do it this way! Thanks for triggering me to look :)